lib/trino-orc/src/test/java/io/trino/orc/OrcTester.java

```diff
@@ -995,6 +996,26 @@ public static DataSize writeOrcColumnHive(File outputFile, RecordWriter recordWr
         return succinctBytes(outputFile.length());
     }
 
+    public static void writeOrcHiveSingleStringColumn(
```

Comment:
We can probably inline this method on `TestCachingOrcDataSource` and squash this commit in the previous commit.

Reply:
seems, this commit (`Use multi column ORC file generator for TestCachingOrcDataSource`) is obsolete due to it was logically replaced with `Add ORC agnostic column amount write file method`. fixed - i've dropped the commit.